src/dotnet/Azure.ClientSdk.Analyzers/readme.md

```diff
@@ -0,0 +1,7 @@
+## How to test WIP Azure.ClientSdk.Analyzers against azure-sdk-for-net

```

Comment:
Thank you for this, by the way - I am using this guidance today, and very grateful to have it!